API.md

```diff
@@ -412,24 +412,33 @@ to Mixpanel.
 - Headers: Authorization: Bearer \<JWT from FusionAuth> (can be user or admin)
 - Request Body
 
+See
+[database](https://github.com/PermanentOrg/back-end/blob/main/library/db/migrations/20240213011017_add_event_table.sql)
```

Comment:
Probably better to link to [Confluence](https://permanent.atlassian.net/wiki/spaces/EN/pages/2118975490/Events+Schema) for the list of actions and entities, since as we add new ones that migration won't reflect them.

Reply:
Thank you! Updated.